### ISSUE TITLE
Custom trigger clarity icon is not reachable with tab key navigation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4655]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.3 Focus order

**Test Environment:**
- Test URL: https://clarity.design/documentation/signposts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. open the test URL.
2. Using tab key navigation try to move focus onto custom trigger clarity icon.

**Expected result:**
Both - Keyboard as well as screen reader user should be able to move focus onto the custom trigger clarity icon with tab key navigation.

**Actual result:**
Focus is skipped from custom trigger clarity icon. Screen reader users can still manage to move focus onto the icon using arrow key navigation but keyboard users cannot navigate on it as tab key navigation is the only way they can interact with the clickable icon.
